src/lib/hooks/use-on-click-outside.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react'
```

Comment:
nit: Wanna move to `src/hooks`? Per https://github.com/hashicorp/dev-portal#component-organization

Reply:
Done, thanks for flagging!